dgl/src/Window.cpp

```diff
@@ -203,6 +213,14 @@ struct Window::PrivateData {
             return;
         }
 
+#if defined(HAVE_DGL)
```

Comment:
This part does not look right.
We will have to think of a way to pick either GL or Cairo.
Do we want it to be a compile-time option? or dynamic at runtime?
In case both are available during build time, which ones has priority?
(these are questions for myself too)

Reply:
I had compile-time choice in mind, but that would sound like a possibility.
I know some users will enjoy a fallback on software rendering when GL support is defective on old machines.